stor/obs.py

```diff
@@ -272,6 +272,10 @@ def __iter__(self):
         # give users access to underlying buffer)
         return self
 
+    def __del__(self):
+        if hasattr(self, '_buffer') and not self.closed:
```

Comment:
should the ` if hasattr(self, '_buffer')` condition actually be in `def close()` ?